data/ships.txt

```diff
@@ -171,30 +171,30 @@ ship "Bactrian"
 	attributes
 		category "Heavy Warship"
 		"cost" 17600000
-		"shields" 17500
-		"hull" 8600
+		"shields" 12400
+		"hull" 9800
 		"required crew" 70
 		"bunks" 245
 		"mass" 940
 		"drag" 16.1
 		"heat dissipation" .4
 		"fuel capacity" 700
-		"cargo space" 530
-		"outfit space" 740
-		"weapon capacity" 300
-		"engine capacity" 180
+		"cargo space" 240
```

Comment:
Another important point of having the max outfit capacity at 900, this means that alien ships actually make sense to upgrade to. 

(Balanced) Bactrian: 900
Shield Beetle: 903
Solifuge: 933
Model 256: 902
Model 512: 1,069
Kar Ik Vot: 1,114
Hurricane: 1,020
Heliarch Punisher:  945

Only exceptions being these two
Raider: 841
Derecho: 826

(Current) Bactrian: 1,130 <- Way too high
(Messymix's) Bactrian: 915 <- Just a little bit too high